moztelemetry/heka/message_parser.py

```diff
@@ -35,7 +38,10 @@ def _parse_heka_record(record):
             # because it contains null values for JSON fields that have been
             # split out.
             if field.name == 'submission':
-                payload = _parse_json(field.value_bytes[0].decode('utf-8'))
+                try:
+                    payload = _parse_json(field.value_bytes[0].decode('utf-8'))
```

Comment:
We probably don't need to do this, since `message.payload` is only ever populated by something that is guaranteed to be (escaped) UTF8.